example/Cargo.toml

```diff
@@ -11,3 +11,5 @@ edition = "2021"
 #momento = "0.15.0"
 momento = { path = "../" }
```

Comment:
examples are currently using the local SDK, but once we cut a release we can update this here